mycore-pi/src/main/java/org/mycore/pi/urn/rest/MCRURNGranularRESTService.java

```diff
@@ -269,6 +273,36 @@ protected void update(MCRDNBURN identifier, MCRBase obj, String additional)
         LOGGER.info("No update in this implementation");
     }
 
+    @Override
+    public void updateFlag(MCRObjectID id, String additional, MCRPI mcrpi) {
+        MCRBase obj = MCRMetadataManager.retrieve(id);
+        MCRObjectService service = obj.getService();
+        ArrayList<String> flags = service.getFlags(MCRPIService.PI_FLAG);
+        Gson gson = getGson();
+
+        //just update flag for derivate, where additional is ""
+        if(additional.equals("")){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=4992058)